CMakeLists.txt

```diff
@@ -1,13 +1,10 @@
 cmake_minimum_required(VERSION 3.10)
 
-project(espressopp)
+project(espressopp LANGUAGES CXX)
```

Comment:
Travis-CI fails with
```
CMake Error at /usr/share/cmake-3.10/Modules/CheckIncludeFiles.cmake:69 (try_compile):
  Unknown extension ".c" for file
    /home/espressopp/espressopp/build/CMakeFiles/CMakeTmp/CheckIncludeFiles.c
  try_compile() works only for enabled languages.  Currently these are:
    CXX
```
Looks like C has to be enabled as well.

Reply:
I removed the unneeded checks to fix that.